examples/benchmark/configuration/checkpointing.yaml

```diff
@@ -0,0 +1,14 @@
+defaults:
+  - dataloader: s3iterabledataset
+  - dataset: unsharded_dataset
+  - training: vit
+  - checkpoint: ???
+  - _self_
```

Comment:
What does this mean?

Reply:
It controls the override behavior, though looking at [the docs](https://hydra.cc/docs/advanced/defaults_list/#composition-order), it appears adding `_self_` at the bottom is the default behavior making this redundant. I'll omit this in the next revision.